must-gather/collection-scripts/gather_ceph_resources

```diff
@@ -250,12 +250,23 @@ for ns in $namespaces; do
         blockpools=$(timeout 60 oc get cephblockpools.ceph.rook.io -n openshift-storage -o jsonpath="{range .items[*]}{@.metadata.name}{'\n'}{end}")
         for bp in $blockpools; do
             images=$(timeout 60 oc -n openshift-storage exec "${HOSTNAME}"-helper -- bash -c "rbd ls -p $bp")
+            pids_rbd=()
             for image in $images; do
-                { timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "rbd info $image --pool $bp" >> "${COMMAND_OUTPUT_FILE}"; } >> "${COMMAND_ERR_OUTPUT_DIR}"/gather-rbd-image-"${image}"-debug.log 2>&1 &
-                pids_ceph+=($!)
-                { timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "rbd snap ls --all $image --pool $bp" >> "${COMMAND_OUTPUT_FILE}"; } >> "${COMMAND_ERR_OUTPUT_DIR}"/gather-rbd-snap-"${image}"-debug.log 2>&1 &
-                pids_ceph+=($!)
+                printf "collecting vol and snapshot info for %s\n" "${image}" | tee -a  "${BASE_COLLECTION_PATH}"/gather-ceph-debug.log
+                COMMAND_OUTPUT_FILE_VOL=${COMMAND_OUTPUT_DIR}/rbd_vol_and_snap_info_${image}.part
+                { printf "Collecting image info for: %s\n" "${image}" >> "${COMMAND_OUTPUT_FILE_VOL}";
+                timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "rbd info $image --pool $bp" >> "${COMMAND_OUTPUT_FILE_VOL}"; >> "${COMMAND_ERR_OUTPUT_DIR}"/gather-rbd-image-"${image}"-debug.log 2>&1
```

Comment:
CI fails with a shellcheck error: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/red-hat-storage_ocs-operator/1611/pull-ci-red-hat-storage-ocs-operator-main-ocs-operator-ci/1519985583051509760#1:build-log.txt%3A67

The `;` between `"${COMMAND_OUTPUT_FILE_VOL}"; >> "${COMMAND_ERR_OUTPUT_DIR}"` is not needed.